### PR TITLE
General improvements.

### PR DESCRIPTION
**Pull request by LTurret**
- Add license description. https://github.com/NFU-OpenDataStructure/NFU-DS-ReferenceSolution/commit/a6ffc99512a466568bc9852417305a44b4bd4cde
- Exclude `.vscode` from repository. https://github.com/NFU-OpenDataStructure/NFU-DS-ReferenceSolution/commit/4142a6cba7c215b0f9bd0e8f0f01c6a8a19d0df9
- Structure rebuild. https://github.com/NFU-OpenDataStructure/NFU-DS-ReferenceSolution/commit/a6ffc99512a466568bc9852417305a44b4bd4cde, https://github.com/NFU-OpenDataStructure/NFU-DS-ReferenceSolution/commit/a6ffc99512a466568bc9852417305a44b4bd4cde
- Reformat annotation codes. https://github.com/NFU-OpenDataStructure/NFU-DS-ReferenceSolution/commit/8d93b4981e057a1f82cb9de9860999d25856e635
- Rewrite some docs for better understanding. https://github.com/NFU-OpenDataStructure/NFU-DS-ReferenceSolution/commit/8d93b4981e057a1f82cb9de9860999d25856e635

### CODE REVIEW

**Comment by LTurret** [comment]
# TODO

- [x] 第一天的部分解答
- [x] 第二天的解答

**Comment by LTurret** [comment]
# TL;DR

1. General improvements.
2. Reconstruct.
3. New contents
4. More solution.

Solution author: @kiol1812
